controllers/reconcilers/grafana/ingress_reconciler.go

```diff
@@ -32,19 +32,19 @@ func NewIngressReconciler(client client.Client, isOpenShift bool) reconcilers.Op
 	}
 }
 
-func (r *IngressReconciler) Reconcile(ctx context.Context, cr *v1beta1.Grafana, status *v1beta1.GrafanaStatus, vars *v1beta1.OperatorReconcileVars, scheme *runtime.Scheme) (v1beta1.OperatorStageStatus, error) {
+func (r *IngressReconciler) Reconcile(ctx context.Context, cr *v1beta1.Grafana, _vars *v1beta1.OperatorReconcileVars, scheme *runtime.Scheme) (v1beta1.OperatorStageStatus, error) {
```

Comment:
The Go convention is not to prefix unused variables, but rather use `_` (`_vars` -> `_`).